<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>binary</class>
 <widget class="QWidget" name="binary">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>992</width>
    <height>633</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,0">
   <property name="leftMargin">
    <number>5</number>
   </property>
   <property name="rightMargin">
    <number>5</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="WidgetVisualize" name="widget_vis" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Labeling</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,3">
      <item>
       <layout class="QFormLayout" name="formLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Scores</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="CheckableComboBox" name="comboBox_score"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Labeling</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QPushButton" name="pushButton_start">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Start</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QProgressBar" name="progressBar">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="value">
             <number>24</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QWidget" name="widget_label_keys" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,1,4,1,0">
         <property name="leftMargin">
          <number>9</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_6">
           <item>
            <widget class="QPushButton" name="pushButton_fast_prev">
             <property name="toolTip">
              <string>Skip to last unlabeled event</string>
             </property>
             <property name="text">
              <string>&lt;&lt;</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="spinBox_jump_to">
             <property name="toolTip">
              <string>Event index, edit to jump to event</string>
             </property>
             <property name="minimum">
              <number>1</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QPushButton" name="pushButton_prev">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>&lt; previous</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_score_prev">
             <property name="text">
              <string>(GMC)</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="layout_label_buttons"/>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QPushButton" name="pushButton_next">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>next &gt;</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_score_next">
             <property name="text">
              <string>(RBC)</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QPushButton" name="pushButton_fast_next">
             <property name="toolTip">
              <string>Skip to next unlabeled event</string>
             </property>
             <property name="text">
              <string>&gt;&gt;</string>
             </property>
            </widget>
           </item>
           <item alignment="Qt::AlignRight|Qt::AlignBottom">
            <widget class="QToolButton" name="toolButton_reset">
             <property name="toolTip">
              <string>Reset labels for current event</string>
             </property>
             <property name="text">
              <string>T</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>WidgetVisualize</class>
   <extends>QWidget</extends>
   <header>dctag.gui.widget_vis</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CheckableComboBox</class>
   <extends>QComboBox</extends>
   <header>dctag.gui.tab_multiple</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
